arm/Microsoft.ContainerRegistry/registries/webhooks/README.md

```diff
@@ -0,0 +1,27 @@
+# Container Registry Webhooks
```

Comment:
Please remove table autoformatting as per the other readme, plus please rename the file to lowercase `readme.md`

Reply:
If you are looking for the setting to disable:

https://github.com/Azure/ResourceModules/blob/11ecea725adfa6a585de12344389eb0c4e9876f5/.vscode/settings.json#L24

Reply:
This its more about generating the readme with the script. I'll fix that so we can progress on the PR.

Reply:
Regenerated readme with new script